.github/workflows/main.yaml

```diff
@@ -10,9 +14,14 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install -y --no-install-recommends doxygen doxygen-latex
+    - name: Get HPCX
+      run: wget "${HPCX_PKG_LINK}/${HPCX_PKG_NAME}.tbz" -P /tmp
```

Comment:
Do we really need HPCX now? Maybe git clone and build only ucx instead?

Reply:
done. Building against v1.9. I don't think we want to use master (smth might change easily in ucx/master and it may break our build here)